pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -449,6 +449,14 @@ type TiKVSpec struct {
 	// +optional
 	SeparateRaftLog *bool `json:"separateRaftLog,omitempty"`
 
+	// Optional volume name configuration for rocksdb log.
+	// +optional
+	RocksDBLogVolumeName string `json:"rocksDBLogVolumeName,omitempty"`
```

Comment:
Please validate this in pkg/apis/pingcap/v1alpha1/validation/validation.go, refer to https://github.com/pingcap/tidb-operator/pull/3731

Reply:
thanks, fixed.